central_v2/src/components/cards/createquestion/CreateQuestionCardBase.tsx

```diff
@@ -111,13 +112,19 @@ export default function CreateQuestionCardBase({
   isCardErrored,
   isAIError,
   isPublic,
+  isCreateGame,
 }: CreateQuestionCardBaseProps) {
   const theme = useTheme();
   const { imageUrl, image } = draftQuestion.questionCard;
   const [questionType, setQuestionType] = React.useState<PublicPrivateType>(PublicPrivateType.PUBLIC);
   const [isImageHovered, setIsImageHovered] = React.useState<boolean>(false);
   const [CCSSIsOpen, setCCSSIsOpen] = React.useState<boolean>(false);
 
+  const isCreateGamePage = 
```

Comment:
Do we need this? it seems like it will always just reflect isCreateGame (as it is looking for all ScreenSize options)